ros/src/computing/planning/decision/packages/decision_maker/nodes/decision_maker/decision_maker_node_state_vehicle.cpp

```diff
@@ -0,0 +1,145 @@
+#include <decision_maker_node.hpp>
+
+namespace decision_maker
+{
+void DecisionMakerNode::entryInitState(cstring_t& state_name, int status)
+{
+  ROS_INFO("Hello Autoware World");
+
+  ROS_INFO("ROS is ready");
+
+  tryNextState("init_start");
+}
+
+void DecisionMakerNode::updateInitState(cstring_t& state_name, int status)
+{
+  static bool is_first_callback = true;
+
+  if (!is_first_callback)
+  {
+    return;
+  }
+  ROS_INFO("Autoware is initializing now");
+  is_first_callback = false;
+}
+
+void DecisionMakerNode::entrySensorInitState(cstring_t& state_name, int status)
+{
+  Subs["filtered_points"] = nh_.subscribe("filtered_points", 1, &DecisionMakerNode::callbackFromFilteredPoints, this);
+  publishOperatorHelpMessage("Please publish \"filtered_points\"");
+}
+
+void DecisionMakerNode::updateSensorInitState(cstring_t& state_name, int status)
+{
+  const double timeout = 1;
+
+  std::vector<std::string> node_list;
+
+  ros::master::getNodes(node_list);
+  for (const auto& i : node_list)
+  {
+    if ("/wf_simulator" == i)
+    {
+      setEventFlag("on_mode_sim", true);
+    }
+  }
+
+  if (isEventFlagTrue("on_mode_sim"))
+  {
+    ROS_INFO("DecisionMaker is in simulation mode");
+    tryNextState("sensor_is_ready");
+  }
+  else if (isEventFlagTrue("received_pointcloud_for_NDT"))
+  {
+    tryNextState("sensor_is_ready");
+  }
+  ROS_INFO("DecisionMaker is waiting filtered_point for NDT");
```

Comment:
Should this message be displayed even in "on_mode_sim"?

Reply:
Fixed not to display if on_mode_sim is true.